### PR TITLE
Fix/wtr732 padding pax page

### PR DESCRIPTION
**Pull request by tomashq**

### CODE REVIEW

File: api/_lib/config.js [hunk 1]
```diff
@@ -78,7 +78,8 @@ const STRIPE_CONFIG =
         PUBLISHABLE_KEY: getConfigKey('STRIPE_PUBLISHABLE_KEY'),
         SECRET_KEY: getConfigKey('STRIPE_SECRET_KEY'),
         WEBHOOK_SECRET: getConfigKey('STRIPE_WEBHOOK_SECRET'),
-        BYPASS_WEBHOOK_SIGNATURE_CHECK: (getConfigKey('STRIPE_BYPASS_WEBHOOK_SIGNATURE_CHECK',"no") === "yes"),
```
Comment (mtahon) [inline api/_lib/config.js:81]: @tomashq please revert this before pushing to `develop`

File: api/_lib/shopping-cart.js [hunk 2]
```diff
@@ -288,13 +293,31 @@ class ShoppingCart {
...
+
+
 
 }
 
 module.exports = {
     ShoppingCart,
     CART_ITEMKEYS,
     CART_USER_PREFERENCES_KEYS,
+    BOOKEABLE_ITEMS_IN_CART
```
Comment (mtahon) [inline api/_lib/shopping-cart.js:320]: typo: `BOOKEABLE_ITEMS_IN_CART` => `BOOKABLE_ITEMS_IN_CART` [source](https://dictionary.cambridge.org/dictionary/english/bookable)

File: api/cart/reprice.js [hunk 3]
```diff
@@ -92,8 +92,9 @@ const offerRepriceController = async (req, res) => {
             // masterOffer.suboffers.push(cart.items[CART_ITEMKEYS.ACCOMMODATION_OFFER]);
         }
 
-
+        cart.totalPrice.public=10;
```
Comment (mtahon) [inline api/cart/reprice.js:95]: Hardcode to be removed before merge

**Review by mtahon — CHANGES_REQUESTED** [review]

File: api/cart/reprice.js (hunk 3 above)
Comment (tomashq) [reply api/cart/reprice.js]: done

File: api/_lib/shopping-cart.js (hunk 2 above)
Comment (tomashq) [reply api/_lib/shopping-cart.js]: done

File: api/_lib/config.js (hunk 1 above)
Comment (tomashq) [reply api/_lib/config.js:81]: done

**Pull request merged**